example/lib/shapes/shadow.dart

```diff
@@ -0,0 +1,165 @@
+import 'dart:math';
+
+import 'package:flutter/widgets.dart';
+import 'package:swift_ui/swift_ui.dart';
+
+class ShadowDemo extends StatelessWidget {
+  const ShadowDemo({
+    super.key,
+  });
+
+  static const color = Colors.grey;
+  static const radius = 10.0;
+  static const x = 10.0;
+  static const y = 10.0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      color: Colors.white,
+      child: const VStack(
+        [
+          // Rectangle with shadow
+          Frame(
+            width: 200,
+            height: 100,
+            child: Shadow(
+              color: color,
+              radius: radius,
+              x: x,
+              y: y,
+              child: Rectangle(
+                fillColor: Colors.purple,
+              ),
+            ),
+          ),
+
+          // Ellipse with shadow
+          Frame(
+            width: 200,
+            height: 100,
+            child: Shadow(
+              color: color,
+              radius: radius,
+              x: x,
+              y: y,
+              child: Ellipse(
+                fillColor: Colors.yellow,
+              ),
+            ),
+          ),
+
+          // Star with shadow
+          Frame(
+            width: 200,
+            height: 200,
+            child: Shadow(
+              color: color,
+              radius: radius,
+              x: x,
+              y: y,
+              child: Star(
+                points: 5,
+                strokeLineWidth: 5,
+                strokeColor: Colors.blue,
+              ),
+            ),
+          ),
+
+          // stroked ellipse
+          Frame(
+            width: 200,
+            height: 100,
+            child: Shadow(
+              color: color,
+              radius: radius,
+              x: x,
+              y: y,
+              child: Ellipse(
+                strokeColor: Colors.red,
+                strokeLineWidth: 5,
+              ),
+            ),
+          ),
+        ],
+        spacing: 20,
+      ),
+    );
+  }
+}
+
+class Star extends StatelessWidget {
```

Comment:
It looks like `Star` is part of the Swift UI API, right? If so, shouldn't we implement `Star`, too, instead of creating a one-off example implementation?

Reply:
No, `Star` isn't a part of SwiftUI as far as I know. I used an LLM to implement the star in both SwiftUI and Flutter because I needed a non-rectangular shape to demonstrate how shadows are applied.